CHANGELOG.md

```diff
@@ -524,7 +529,10 @@ Functionally the same as [0.17.3] but with some CI updates.
 ### Added
 - Initial release!
 
-[Unreleased]: https://github.com/OSC/ood_core/compare/v0.23.4...HEAD
+[Unreleased]: https://github.com/OSC/ood_core/compare/v0.24.2...HEAD
+[0.24.2]: https://github.com/OSC/ood_core/compare/v0.24.2...v0.24.1
+[0.24.1]: https://github.com/OSC/ood_core/compare/v0.24.1...v0.24.0
+[0.24.0]: https://github.com/OSC/ood_core/compare/v0.24.0...v0.23.5
```

Comment:
Are those version numbers at the end right? My brain wants these the other way `v0.23.5...v0.24.0` etc. Am I reading that wrong?

Reply:
Yep! You're right, they're backwards.

Reply:
and oddly enough, it was just that one link.

Reply:
Wait, now I'm all mixed up. It should be `lower...higher`. They're links, so they should actually link to something correct. I'm validating them all now.

Reply:
OK - I think they're all correct now.